File: cs/proxy/hall/ActivityOperation.cs
/// <summary>
/// 
/// </summary>
namespace org.zgl.service.server.hall{
	[Proxy(gameId = -10)]
	public interface ActivityOperation : ITcpAsyncService { 
		/// <summary>
		/// 
		/// </summary>
		void pay(int productId);
		/// <summary>
		///  ----> <<pay();回调>>
		/// </summary>
		void pay2CallBack();
	}
}

File: componet/src/main/java/org/zgl/service/server/hall/ActivityOperation.java
package org.zgl.service.server.hall;

import org.zgl.proxy.rule.ITcpAsyncService;

/**
 * @author： 猪哥亮
 * @创建时间： 2018/8/21
 * @文件描述：活动
 */
public interface ActivityOperation extends ITcpAsyncService {
    void pay(int productId);
}
